README.md

```diff
@@ -1 +1,22 @@
-# rmb-sdk-ts
\ No newline at end of file
+# Introduction
+This is a `Typescript` sdk that can be used to build both **services**, and **clients**
+that can talk over the `rmb`.
+
+[RMB](https://github.com/threefoldtech/rmb-rs) is a message bus that enable secure
+and reliable `RPC` calls across the globe.
+
+`RMB` itself does not implement an RPC protocol, but just the secure and reliable messaging
+hence it's up to server and client to implement their own data format.
+
+## Pre Requirements
+You need to run an `rmb-relay` locally. 
+
+### Direct Client
```

Comment:
For simplicity let's just call it RMB client instead and not mention the otherflow completely, maybe in a separate page instead

Reply:
done